### ISSUE TITLE
add URI of repository to the repository section 

### ISSUE THREAD

**Issue by fabianneuhaus** [tillmo/DOL#291]

**Comment by tillmo** [comment]
Probably you mean the registry? Originally, there was the URL http://purl.net/DOL/registry mentioned as URL of the registry. @fabianneuhaus, you have deleted this and added a paragraph starting "OMG hosts a registry for DOL-conforming languages and translations." Does this mean that http://purl.net/DOL/registry is not appropriate and that there should be an OMG URL for the registry?

**Comment by fabianneuhaus** [comment]
Yes, I meant registry. -- We will need to add the official URI when we know it. I pushed this to the FTF.
